app/views/jobs/_ganglia_graphs_table.html.erb

```diff
@@ -0,0 +1,28 @@
+<table>
+<div class="panel panel-default">
+	<tr class="col-xs-12">
+		<div class="col-xs-12">
+   			<tr><div class="job_data" data-jobid="<%= d.pbsid %>" ></div></tr>
+  		</div>
+    </tr>
+	<% if !d.nodes.nil? && has_ganglia(d.cluster) %>
+		<% d.nodes.each do |node| %>
+			<div class="col-md-12">
+				<tr>
+					<td>
+						<div class="panel panel-default">
+							<div class="panel-heading"><h3 class="panel-title"> <%= node %> <span class="pull-right"> <%= d.pbsid %> </span></h3></div>
+							<div class="panel-body">
+								<%= link_to  image_tag( build_ganglia_link(d.cluster, d.starttime, 'cpu_report', node, 'small'), class:"img-responsive col-lg-3 col-md-3 col-sm-6 col-xs-6" ), build_ganglia_link(d.cluster, d.starttime, 'cpu_report', node, 'large'), data: { lightbox: "cpu-report", title: "CPU Report " + node } %>
+								<%= link_to  image_tag( build_ganglia_link(d.cluster, d.starttime, 'load_report', node, 'small'), class:"img-responsive col-lg-3 col-md-3 col-sm-6 col-xs-6" ), build_ganglia_link(d.cluster, d.starttime, 'load_report', node, 'large'), data: { lightbox: "load-report", title: "Load Report " + node } %>
+								<%= link_to  image_tag( build_ganglia_link(d.cluster, d.starttime, 'mem_report', node, 'small'), class:"img-responsive col-lg-3 col-md-3 col-sm-6 col-xs-6" ), build_ganglia_link(d.cluster, d.starttime, 'mem_report', node, 'large'), data: { lightbox: "mem-report", title: "Memory Report " + node } %>
+								<%= link_to  image_tag( build_ganglia_link(d.cluster, d.starttime, 'network_report', node, 'small'), class:"img-responsive col-lg-3 col-md-3 col-sm-6 col-xs-6" ), build_ganglia_link(d.cluster, d.starttime, 'network_report', node, 'large'), data: { lightbox: "network-report", title: "Network Report " + node } %>
+							</div>
+           				</div>
+           			</td>
+           		</tr>
+           	</div>
+        <% end %>
+	<% end %>

```

Comment:
I'd actually like to see this change to a partial that is _job_details_node_view.

That would mean you would render the partial over each node, with the node and the cluster as local arguments to the partial. In the _job_details_node_view there could be the check for <% if has_ganglia %>.

The idea being that this would enable us and others to override just this per node view with a custom per-site view.

Reply:
Clever idea, I have just committed and pushed your requests. Please review them and let me know if there is anything that needs to be changed or edited. Thanks.

Reply:
Don't use single letters for variable names. So in the partial's case, ":node => node" is better than ":n => node".

Reply:
Got it. Thanks.